lib/Report/Transform/SuiteCollectionTransformer.php

```diff
@@ -13,12 +14,9 @@ final class SuiteCollectionTransformer
 {
     public const COL_HAS_BASELINE = 'has_baseline';
 
-    /**
-     * @return array<string,array<string,mixed>>
-     */
-    public function suiteToTable(SuiteCollection $collection, bool $includeBaseline = false): array
+    public function suiteToTable(SuiteCollection $collection, bool $includeBaseline = false): DataFrame
```

Comment:
this is a BC break potentially - add another method and deprecate the `array` one